mta/fixtures/application_inventory.py

```diff
@@ -3,50 +3,36 @@
 import pytest
 
 from swagger_client.models.api_application import ApiApplication
-from swagger_client.models.api_task import ApiTask
 
 
-@pytest.fixture(scope="session")
 def json_application():
     with open("mta/data/application.json", "r") as file:
-        yield json.load(file)
+        return json.load(file)
 
 
 @pytest.fixture(scope="session")
-def analyses_data():
+def applications_data():
+    return json_application()
+
+
+def json_analysis():
     with open("mta/data/analysis.json", "r") as file:
         return json.load(file)
 
 
+@pytest.fixture(params=json_analysis(), ids=[item.get("source") for item in json_analysis()])
+def analysis_item(request):
```

Comment:
maybe change the name to something like analysis_item_matrix?

Reply:
@amastbau, based on your comment I modified the test to use `@pytest.mark.parametrize` instead of fixture:
`@pytest.mark.parametrize("analysis_item", json_analysis(), ids=[item.get("source") for item in json_analysis()])`
Please let me know what you think about the parameter's name (analysis_item)